packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -26,6 +36,8 @@ export interface IShortcutItemProps {
   clearConflicts: Function;
   contextMenu: Function;
   external: IShortcutUIexternal;
+  tabIndex: number;
+  handleRowKeyDown: Function;
```

Comment:
`Function` will be disallowed going forward, see https://github.com/jupyterlab/jupyterlab/issues/16038#issuecomment-2015398728.

Reply:
Thanks for letting us know, to be able to refactor is dependent on your PR being merged. As mentioned previously, without being able to work on this past 28/03/24 would you be happy to take care of the refactor once merged?